pkg/kubelet/kubelet_getters.go

```diff
@@ -225,7 +226,7 @@ func (kl *Kubelet) getRuntime() kubecontainer.Runtime {
 // GetNode returns the node info for the configured node name of this Kubelet.
 func (kl *Kubelet) GetNode() (*v1.Node, error) {
 	if kl.kubeClient == nil {
-		return kl.initialNode()
+		return kl.initialNode(context.TODO())
```

Comment:
What's TODO here? Is the idea to plumb the context through as an arg to `GetNode()`? Or should this just be `context.Background()`?

Reply:
My thought was that yes, it would get plumbed through GetNode(), but I'm trying to handle this in small chunks. I started trying to do this more globally with https://github.com/kubernetes/kubernetes/pull/58532 and gave up as it seemed too much to review. So now I'm trying to wire it through in stages.